### PR TITLE
Add generated docs as markdown files under /doc

### PR DESCRIPTION
**Pull request by bjosv**
Generating the documentation as-is, but it certainly needs to be reviewed in a near future.

Also correcting an indent flaw in CI file

### CODE REVIEW

**Comment by bjosv** [comment]
hum, no Ci..

**Comment by zuiderkwast** [comment]
Nice, but before checking in the docs, I think we should mark all internal modules as `@private` and delete the unused erldis file. If there are exported functions which should not be part of public API, also mark them `@private`.

**Comment by zuiderkwast** [comment]
Btw, the commit message says "/docs" but it is "/doc".

File: rebar.config [hunk 1]
```diff
@@ -42,3 +42,18 @@
                , {erldis, set, 3}
                , {erldis, mget, 2}
                ]}.
+
```
Comment (bjosv) [inline rebar.config:45]: Remove erldis in:
`
%% basho_bench_driver_erldis calls undefined functions that we ignore.
{xref_ignores, [ {erldis_client, connect, 0}
               , {erldis, set, 3}
               , {erldis, mget, 2}
               ]}.
`
and erldis in
`cover_excl_mods, [ basho_bench_driver_erldis`

**Review by bjosv — COMMENTED** [review]
and remove config file:
priv/basho_bench_erldis.config

**Review by bjosv — COMMENTED** [review]
LGTM

**Pull request merged**